bump dotnet and add minver (#13)
aserto-dev/aserto-dotnet: README.md

| Enabled | true | Enables or disables Aserto Authorization |
| ServiceUrl | "https://authorizer.prod.aserto.com:8443" | Sets the URL for the authorizer endpoint. |
| Decision | "allowed" | The decision that will be used by the middleware when creating an authorizer request. |
| AuthorizerApiKey | "" | The authorizer API Key |

> | TenantID | "" | Te Aserto Tenant ID |

Reviewer: nit `The`

| Inscure | false | Indicates whether insecure service connections are allowed when using SSL |
| PolicyName | "" | The Aserto policy name |

> | PolicyInstanceLabel | "" | instance label that will be used for all the requests |

Reviewer 2: The description for policy name and instance label should be updated

> | PolicyInstanceLabel | "" | instance label that will be used for all the requests |

Reviewer 2: ```suggestion
| PolicyInstanceLabel | "" | The label of the active policy runtime |
```
